oxidation/crates/parsec_api_crypto/src/rustcrypto/hash.rs

```diff
@@ -46,8 +45,9 @@ impl TryFrom<&[u8]> for HashDigest {
     type Error = CryptoError;
     fn try_from(data: &[u8]) -> Result<Self, Self::Error> {
         // if you wonder, `try_into` will also fail if data is too small
```

Comment:
You remove this comment in the other methods but not here, why ?

(I guess you can just remove it)

Reply:
Because this is not the same try_into, even if it implements